src/main/scala/org/globalforestwatch/summarystats/SummarySparkSession.scala

```diff
@@ -24,6 +25,9 @@ object SummarySparkSession {
         SparkSession.builder.config(localConf).getOrCreate
       case e: Throwable => throw e
     }
+
+    GeoSparkSQLRegistrator.registerAll(spark)
```

Comment:
Not sure what this does, enable all the spatial function in for Spark SQL?

Reply:
Yeah, this adds the GeoSpark extension methods/types to Spark SQL: https://datasystemslab.github.io/GeoSpark/tutorial/sql/#register-geosparksql